pkg/gadgets/advise/seccomp/tracer/tracer.go

```diff
@@ -125,6 +125,10 @@ func (t *Tracer) Delete(mntns uint64) {
 }
 
 func (t *Tracer) Close() {
+	t.cleanup()
+}
+
+func (t *Tracer) cleanup() {
```

Comment:
I'm not totally convinced about calling it `cleanup()`, if you look at the implementations around, all of them call `*Close()` methods, so why not to call it `close()`? I know in some cases there will be `Close()` and `close()`, but the first one is temporal and will be removed later on. WDYT?

Reply:
I tried to change it from `Close` to `Cleanup` because I want to avoid overlapping between gadgets implementing `RunGadget`/`RunWithResultGadget` with gadgets implementing `InitRunClose` (which requires `Close()` to be implemented).

I don't want to go to a `RunGadget` and see the `Run` and `Close` functions and think _"Where is the `Init`, is it missing?"_. WDYT?

Reply:
> I don't want to go to a RunGadget and see the Run and Close functions and think "Where is the Init, is it missing?". WDYT?

IMO it's not a big issue. I think it's better to call all functions that have a similar intention with the same name, otherwise we'll end with `cleanup()`,`close()`, `uninstall()` when all of them serve to the same purpose. 

However I don't have a strong opinion, if you think another naming makes more sense that's fine to me.

Reply:
I moved back to `close()` because it will be unexported and it will not receive the gadget-context as one in the `InitClose` interface will do (see https://github.com/inspektor-gadget/inspektor-gadget/pull/1408#discussion_r1144074019). Therefore, it should be easy for developer to distinguish them.